src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4841,7 +4843,17 @@ def webapp_up(cmd, name=None, resource_group_name=None, plan=None, location=None
 
     if logs:
         _configure_default_logging(cmd, rg_name, name)
-        return get_streaming_log(cmd, rg_name, name)
+        # ping the site so that the site container is ready
+        # logsteam does not work if the site container has not started
+        # required for https://github.com/Azure/azure-cli/issues/23058
+        _url = _get_url(cmd, rg_name, name)
```

Comment:
we can move the `_url = ` and `http = ` also inside the `try` no?

Reply:
updated, thanks!